client/landing/stepper/declarative-flow/link-in-bio.ts

```diff
@@ -43,7 +44,9 @@ export const linkInBio: Flow = {
 					return navigate( 'linkInBioSetup' );
 
 				case 'linkInBioSetup':
-					return window.location.replace( '/start/link-in-bio/domains' );
+					return window.location.replace(
+						`/start/newsletter/domains?new=${ getState().siteTitle ?? '' }&search=yes`
+					);
```

Comment:
This can create unpredictable behavior because we don't know which site we're dealing with. I think you should use the submitted site title from the linkInBioSetup step, `providedDependencies.siteTitle`. And please make sure to `encodeURIComponent`.

Reply:
much better tnx